CHANGES.rst

```diff
@@ -68,6 +68,10 @@ Release history
   values. (`#1609`_)
 - The ``nl`` and ``nl_nodirect`` test arguments have been renamed to ``AnyNeuronType``
   and ``NonDirectNeuronType``. (`#1609`_)
+- Weight solvers (i.e. those with ``weights=True``) now work on connections from
```

Comment:
Probably "are now allowed" rather than "work on", which feels ambiguous as to what it means to "work". If you're consider their previous disallowance as a bug, then this should be in the "fixed" section.